### PR TITLE
Bug 1232588 - wrap Media Storage panel into AMD module, r=yzen

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f3887a521aaa8a4be5eecbbc51c47a3adc289504

File: apps/settings/js/panels/media_storage/media_storage.js [hunk 1]
```diff
@@ -8,14 +6,14 @@
  * In this case, the user has to unplug the USB cable in order to actually turn
  * off UMS, and we put some text to that effect on the settings screen.
  */
-require([
-  'modules/dialog_service',
-  'modules/settings_cache',
-  'modules/storage_helper',
-  'shared/toaster',
-  'shared/settings_listener'
```
Comment (gasolin) [inline apps/settings/js/panels/media_storage/media_storage.js]: settings_listener is not used anymore

File: apps/settings/js/panels/media_storage/panel.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ */
+define(function(require) {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+  var MediaStorage = require('panels/media_storage/media_storage');
+
+  return function ctor_media_storage_panel() {
+    return SettingsPanel({
+      onInit: function(panel) {
```
Comment (yzen) [inline apps/settings/js/panels/media_storage/panel.js]: Nit: panel argument is not used so lets drop it.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d40269ec8e60a3fefcb9a65f817dd3c4552491dd

**Pull request merged**